Networking/NetworkingTests/Models/RefundTests.swift

```diff
@@ -0,0 +1,60 @@
+import XCTest
+@testable import Networking
+
+
+/// Refund Unit Tests
+///
+final class RefundTests: XCTestCase {
+
+    func testFullRefund() {
+
+        let dictExpected: [String: Any] = ["amount": "10.00", "api_refund": true, "reason": "Product No Longer Needed"]
```

Comment:
This is highly subjective, and definitely not a blocker, so you can ignore it, but if I was writing these tests I would probably try to extract the creation of the expectation to a private method, so that this particular test reads like:

````
func testFullRefund() {
	let expectation = dictionaryWithAmountAndReason()
	
	let refund = refundWithAmountAndReason()
	let refunDict = refund.toDictionary()

	XCTAssertEqual(refundDict, expectation)
}
````

It seems to me like it would be shorter and easier to read. What do you think?

Reply:
I think that it's a lot easier to read, and it's the approach used everywhere in the project's tests. I'm going to edit it